### DISCUSSION TITLE
Errors about DB rows and columns

### DISCUSSION THREAD

**Discussion by unc1** [apache/skywalking#8311] [Q&A]
Hi,
I am seeing these sort of errors and am wondering if it's a configuration problem on my end? 

2021-12-17 10:34:46,306 - org.apache.skywalking.oap.server.storage.plugin.jdbc.h2.dao.H2BatchDAO - 75 [pool-5-thread-2] ERROR [] - Data truncation: Data too long for column 'dest_endpoint' at row 2
java.sql.BatchUpdateException: Data truncation: Data too long for column 'dest_endpoint' at row 2

Any insight would be great. I am using the latest Skywalking and MYSQL. Thanks.

**Comment by wu-sheng** [comment]
Read #8272 We are not sure which characters in the ID makes length control mechanism fails. Need your local debug to verify.

**Reply by unc1** [reply]
I'll figure out how to get the debugging turned on, but the only characters outside of (a-z)(0-9) being used are  (.) and a (-).

**Reply by wu-sheng** [reply]
Yes, because it is base64 encoded.  The length controlling is for raw string. Maybe there are some gaps.

**Comment by unc1** [comment]
How would I go about configuring the debug log level? Is it just a JVM argument I supply or configuration file change? Thanks.

**Reply by wu-sheng** [reply]
Change how to change log level though log4j2.xml.
